### PR TITLE
Standardizing transaction pages

### PR DESCRIPTION
**Pull request by jmensah365**

### CODE REVIEW

**Review by achuggi1 — CHANGES_REQUESTED** [review]
Incorrect target branch for pull request.

**Review by msabas1 — DISMISSED** [review]
LGTM

File: cucumber-selenium-tests/src/test/java/com/skillstorm/PageObjects/TransactionHistoryPage.java [hunk 1]
```diff
@@ -259,14 +259,37 @@ public String setCategory(String category) {
      * @return the text of the first row
      */
     public String confirmCreation() {
```
Comment (achuggi1) [inline cucumber-selenium-tests/src/test/java/com/skillstorm/PageObjects/TransactionHistoryPage.java:271]: Low impact: The 2nd, 3rd, and 4th if statements will never be run.
Suggestion: Reverse the order of the if statements or remove the unnecessary ones.

File: cucumber-selenium-tests/src/test/java/com/skillstorm/PageObjects/TransactionPage.java [hunk 2]
```diff
@@ -187,8 +188,38 @@ public String setCategory(String category) {
     }
 
     public String verifyTransactionDetails() {
```
Comment (achuggi1) [inline cucumber-selenium-tests/src/test/java/com/skillstorm/PageObjects/TransactionPage.java:192]: Low impact: The 2nd, 3rd, and 4th if statements will never be run.
Suggestion: Reverse the order of the if statements or remove the unnecessary ones.

**Review by achuggi1 — CHANGES_REQUESTED** [review]
A few if statements are never run.

**Review by achuggi1 — DISMISSED** [review]
LGTM

**Review by msabas1 — DISMISSED** [review]
LGTM

**Review by msabas1 — APPROVED** [review]
LGTM

**Review by Aldrich19 — APPROVED** [review]

**Review by achuggi1 — APPROVED** [review]
LGTM

**Pull request merged**